Toolbox/GEM/mqb-startup.esd

```diff
@@ -84,3 +84,33 @@ keyValue
 script
    value    sys 1 0x0100 "/eso/hmi/engdefs/scripts/mqb/install_forcedcanim.sh"
    label    "Force replacing of startup screen (no coding needed)"  
+   
+keyValue
+    value   String sys 0x00000000 0
+    label   ""
+    poll    0  
+
+keyValue
+    value   String sys 0x00000000 0
+    label   "Install SSH service."
+    poll    0  
+    
+keyValue
+    value   String sys 0x00000000 0
+    label   "Place a copy of your id_rsa.pub login key in Custom/ folder on SD-card"
+    poll    0  
+
+keyValue
+    value   String sys 0x00000000 0
+    label   "before running to enable password-less login."
+    poll    0 
+    
+keyValue
+    value   String sys 0x00000000 0
+    label   ""
+    poll    0 
+
+script
+   value    sys 1 0x0100 "/eso/hmi/engdefs/scripts/mqb/sshd_install.sh"
+   label    "Install SSHD service"  
+
```

Comment:
The current `Startup` esd is meant to change the startup pictures of a unit. So at least in my logic the placement of this function isn't so good here. I would place this here into `mqb-advanced.esd`.

Reply:
Thanks for the suggestion, I thought there would be a better place, just wasn't sure where